ci/infra/libvirt/terraform.tfvars.json.ci.example

```diff
@@ -1,14 +1,13 @@
 {
-    "libvirt_uri": "qemu:///system",
+    "libvirt_uri": "qemu+ssh://jenkins@node52.cloud.suse.de/system",
```

Comment:
How are the credentials been passed to libvirt?

Reply:
I assume the jenkins user for on CI workers to use the jenkins key pair.

Reply:
OK after some investigations I think the way to pass the custom ssh key to the libvirt URI is by passing is as a parameter of the keyfile within the URI, so something like: `qemu+ssh://jenkins@node52.cloud.suse.de/system?keyfile=<file>`. I find it weird adding a file reference of the client host for a remote URI, but it is the only way I found to use something else than `~/.ssh/id_rsa`.

Reply:
I forgot to mention this: I was using this uri for the qemu connection in the test worker `qemu+ssh://jenkins@node52.cloud.suse.de/system` and it was working, BUT now it is not. I need to check. I remember at some point I setup the ssh-agent, but now even if I do, it is not working.

More strangely, if I test the connection with `virsh` and add the key to `ssh-agent` it works
```
$> eval (ssh-agent)
$> ssh-add /home/jenkins/.ssh/jenkins
$>virsh -c qemu+ssh://jenkins@node52.cloud.suse.de/system
```

Reply:
`virsh -c qemu+ssh://jenkins@node52.cloud.suse.de/system` 
This works for me if I properly set the keys or I use in URI parameter `keyfile` to point to the specific private key.
Now I seeing if I can configure a credentials in Jenkins to use a URI similar to  `qemu+ssh://jenkins@node52.cloud.suse.de/system?keyfile=${jenkins_key}` so we do not depend on certain specific underlaying configuration on the worker.

Reply:
This is weird. Now it works with the uri as you describe. Earlier I tried it and terraform kept asking me for a password (I suppose it was not finding the ssh key). Better to keep :eyes: on this.